tempodb/backend/raw_test.go

```diff
@@ -10,6 +10,11 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+const (
+	TenantID      = "test"
+	StoragePrefix = "test/prefix"
```

Comment:
```suggestion
	tenantID      = "test"
	storagePrefix = "test/prefix"
```

I don't think we should export these constants